### PR TITLE
feat: add account type and visibility to target entity

### LINKED ISSUE #651: Update collector data
**Is your feature request related to a problem? Please describe.**
For each event being collected we need to know:
- [ ] Account type - organization / user
- [ ] Visibility - public / private

### PR DESCRIPTION
**Pull request by zolamk**
## Description
Adds `account type` and `repository visibility` to target entity so that we can use it when collecting events to mixpanel.

## Related issue

Closes #651 

## Type of change

New feature (non-breaking change which adds functionality)

## How was this tested?
Unit tests

## Checklist

- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works 
- [x] I have ran `task check -f` and have no issues

## Code review and merge strategy (ship/show/ask) 

 
 
- [x] Ask: this pull request requires a code review before merge

### CODE REVIEW

File: handler/processors.go [hunk 1]
```diff
@@ -88,10 +90,12 @@ func processCronEvent(log *logrus.Entry, token string, e *ActionEvent) ([]*Targe
...
-			Kind:   kind,
-			Number: *issue.Number,
-			Owner:  owner,
-			Repo:   repo,
+			Kind:        kind,
+			Number:      *issue.Number,
+			Owner:       owner,
+			Repo:        repo,
+			AccountType: issue.GetRepository().GetOwner().GetType(),
+			Visibility:  issue.GetRepository().GetVisibility(),
```
Comment (ferreiratiago) [inline handler/processors.go:98]: I'm not sure if `Visibility` is always present. Can you please make sure? Here's how we do it in the [action](https://github.com/reviewpad/action/blob/main/agent/collector.go#L18).

**Review by ferreiratiago — CHANGES_REQUESTED** [review]

**Review by shay2025 — CHANGES_REQUESTED** [review]
It might make sense to lowercase the account type in order to stay consistent with the other events that are collected and display the account type, e.g., https://github.com/reviewpad/action/blob/main/agent/collector.go#L46 and on the RAAS.

**Review by ferreiratiago — APPROVED** [review]

**Comment by zolamk** [comment]
> Member

I think we can lower it on the raas and action side

**Comment by zolamk** [comment]
> It might make sense to lowercase the account type in order to stay consistent with the other events that are collected and display the account type, e.g., https://github.com/reviewpad/action/blob/main/agent/collector.go#L46 and on the RAAS.

I think we can lower it on the raas and action side

**Comment by shay2025** [comment]
> > It might make sense to lowercase the account type in order to stay consistent with the other events that are collected and display the account type, e.g., https://github.com/reviewpad/action/blob/main/agent/collector.go#L46 and on the RAAS.
> 
> I think we can lower it on the raas and action side

On the RAAS and action side, it is lowered already 🙂

**Pull request merged**